tests/test_libraries.py

```diff
@@ -2339,3 +2339,9 @@ def test_h3(pyi_builder):
     pyi_builder.test_source("""
         import h3
     """)
+
+@importorskip('selectolax')
```

Comment:
There should be two empty lines between test functions - see the linter failure. Apart from that, I think we're good.

Reply:
Fixed